src/eth_plugin_interface.h

```diff
@@ -13,7 +13,8 @@ typedef enum {
     ETH_PLUGIN_INTERFACE_VERSION_2 = 2,
     ETH_PLUGIN_INTERFACE_VERSION_3 = 3,
     ETH_PLUGIN_INTERFACE_VERSION_4 = 4,
-    ETH_PLUGIN_INTERFACE_VERSION_LATEST = 5,
+    ETH_PLUGIN_INTERFACE_VERSION_5 = 5,
+    ETH_PLUGIN_INTERFACE_VERSION_LATEST = 6
```

Comment:
Add the comma to avoid git conflicts :
```suggestion
    ETH_PLUGIN_INTERFACE_VERSION_LATEST = 6,
```